docs/documents/span.dox

```diff
@@ -0,0 +1,10 @@
+/** @class precice::span
+ *
+ * @copyright Copyright Tristan Brindle 2018.
```

Comment:
Do we need to ship a third-party license with this?

Reply:
It's included in the header file. That should suffice, right?

Reply:
After a chat with @uekerman regarding this:
* comment was regarding the licence text itself. The header only states which licence it uses
* This should suffice.